ui/src/importer/importerTaskDetails/ImporterReportHeader.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Grid, Icon } from 'semantic-ui-react';
+import { Link } from 'react-router-dom';
+import { CdsBackOfficeRoutes } from '../../overridden/routes/BackofficeUrls';
+import { ImportedSearch } from './ImportedSearch';
+import { CancelImportTask } from './cancelImportTask';
+
+export class ImporterReportHeader extends React.Component {
+  render() {
+    const { status, isLoading, taskId, onSearch } = this.props;
+    return (
+      <Grid>
+        <Grid.Column floated="left" width={12}>
+          <Button
+            className="default-margin-top"
+            labelPosition="left"
+            icon="left arrow"
+            content="Import other files"
+            loading={status}
+            disabled={status}
```

Comment:
Maybe change this to `disabled` instead of status? It reflects better what the passed property is controlling.